juniper_official/routing/check-isis-statistics.rule

```diff
@@ -491,6 +491,13 @@
                                     }
                                 }
                             }
+                            products EX {
```

Comment:
Can we put sensors also at EX level? Same comment for all 3 rules

Reply:
Updated sensor under EX product for all 3 rules.